packages/runtime/container-runtime/src/summaryManager.ts

```diff
@@ -252,20 +244,25 @@ export class SummaryManager extends TypedEventEmitter<ISummaryManagerEvents> imp
         }
     }
 
+    private checkBypassInitialDelay() {
```

Comment:
nit: Rename to `tryBypassInitialDelay` since this would not just check but also bypass the initial day if conditions are met.